### PR TITLE
Fix overhead drawing

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

**Review by AaronVanGeffen — APPROVED** [review]

**Comment by AaronVanGeffen** [comment]
Looks like this may be a regression in the previous release? In that case, it needs a changelog, too.

File: src/OpenLoco/src/Windows/Construction/OverheadTab.cpp [hunk 1]
```diff
@@ -505,6 +503,8 @@ namespace OpenLoco::Ui::Windows::Construction::Overhead
                 clipped->x += screenPos.x;
                 clipped->y += screenPos.y;
```
Comment (ZehMatt) [inline src/OpenLoco/src/Windows/Construction/OverheadTab.cpp:504]: Can you move those lines to 495, its not really used by anything other than pushRenderTarget.

Comment (duncanspumpkin) [reply src/OpenLoco/src/Windows/Construction/OverheadTab.cpp:504]: No, this is the first line with all the state

Comment (ZehMatt) [reply src/OpenLoco/src/Windows/Construction/OverheadTab.cpp:504]: Hm right

**Pull request merged**